src/main/resources/assets/traverse/lang/zh_cn.json

```
@@ -3,20 +3,20 @@
 	"biome.traverse.autumnal_wooded_hills":"秋木山脉",
 	"biome.traverse.autumnal_woods":"秋木树林",
 	"biome.traverse.cliffs":"悬崖",
-	"biome.traverse.coniferous_forest":"针叶森林",
-	"biome.traverse.coniferous_woodland_hills_biome":"针叶林山",
+	"biome.traverse.coniferous_forest":"冷杉森林",
+	"biome.traverse.coniferous_woodland_hills_biome":"冷杉林山",
 	"biome.traverse.desert_shrubland":"沙漠灌丛带",
-	"biome.traverse.high_coniferous_forest":"高针叶森林",
+	"biome.traverse.high_coniferous_forest":"高冷杉森林",
 	"biome.traverse.lush_swamp":"茂密沼泽",
 	"biome.traverse.meadow":"牧场",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"biome.traverse.meadow":"草地",
```